api/desecapi/renderers.py

```diff
@@ -12,28 +15,10 @@ def render(self, data, media_type=None, renderer_context=None):
 
         if response and response.exception:
             response['Content-Type'] = 'text/plain'
-
             try:
                 return data['detail']
-            except (KeyError, TypeError):
-                pass
-
-            try:
-                details = list(filter(None, [el.get('detail') for el in data]))
-                if details:
-                    return ', '.join(details)
-            except (TypeError, AttributeError):
-                pass
-
-            try:
-                return '; '.join([f'{err.code}: {err}' for err in data])
-            except (TypeError, AttributeError):
-                pass
-
-            raise ValueError('Expected response.data to be one of the following:\n'
-                             '- a dict with error details in response.data[\'detail\'],\n'
-                             '- a list with at least one element that has error details in element[\'detail\'];\n'
-                             '- a list with all elements being ErrorDetail instances;\n'
-                             'but got %s:\n\n%s' % (type(response.data), response.data))
+            except:
+                data = json.loads(json.dumps(data))  # stringify exception objects in potentially nested data structure
+                return yaml.safe_dump(data, default_flow_style=False)
```

Comment:
I think I've seen Django outputting plain-text exception messages before; if this really exists we should use that instead of a home-grown solution. I guess our api somehow breaks the content negotiation of Django and or DRF, which is why we sometimes do not see proper error messages.

Reply:
I did not find anything like that, but the solution now is much simpler than before. So instead of staying with the previous, I'd prefer this, unless we learn about a better way to do it (but I failed).

Reply:
Thanks for the review! All done, ready for next round! :)

Reply:
I guess you're right - it's better than what we have, and I don't have time to investigate what I am referring to above. So let's keep it as is for now.